{"id": "graphscale.errors", "path": "/Users/schrockn/code/schrockn_github/graphscale/graphscale/errors.py", "mtime": 1499123398.0, "size": 1238, "data_mtime": 1499127801.0, "dependencies": ["functools", "typing", "traceback", "builtins"], "suppressed": [], "child_modules": [], "options": {"ignore_missing_imports": true, "ignore_errors": false, "warn_no_return": true, "disallow_untyped_defs": false, "quick_and_dirty": false, "strict_boolean": false, "follow_imports": "silent", "debug_cache": false, "show_none_errors": true, "disallow_untyped_calls": false, "strict_optional_whitelist": null, "check_untyped_defs": false, "strict_optional": false, "warn_return_any": false}, "dep_prios": [5, 5, 20, 5], "interface_hash": "0152102b72df8b142e85ba0c4cff4a71", "version_id": "0.511"}